scripts/install.sh

```diff
@@ -338,6 +338,29 @@ if [[ $enable_self_signed_ca == "true" ]]; then
   rm default/kustomization.yaml.bak
 fi
 
+# watch out when newTag has no non-numeric characters, and gets interpreted as an int, rather than srting
+# sed -i.bak 's/newTag:.*$/newTag: '"$GIT_COMMIT_SHORT"'/' config/manager/kustomization.yaml
+# ++ sed -i.bak 's/newTag:.*$/newTag: 7284872/' config/manager/kustomization.yaml
+# Error: accumulating resources from '../manager':
+#   '/config/manager' must resolve to a file':
+#     couldn't make target for path '/config/manager':
+#       invalid Kustomization:
+#         json: cannot unmarshal number into Go struct field
+#           Image.images.newTag of type string
+# error: no objects passed to apply
+# CLI check:
+#   git log -1 --format=%h --abbrev=7
+#     7284872
+# This happened 7 times as of 2023/12/02:
+#   git log --format=%h --abbrev=7 | grep -E "^[0-9]+$"
+#     7284872
+#     4808804
+#     4746079
+#     0281170
+#     1631706
+#     5293579
+#     4207065
+info "kustomize build default"
```

Comment:
Can you elaborate a bit on this comment, did you mean to include it in `scripts/install.sh`? It looks like a reference to [this line](https://github.com/kserve/modelmesh-serving/blob/45cb31b4978ab03ade140cae69da319496effc04/scripts/deploy/iks/deploy-mm-serving.sh#L78).

Reply:
Right, this an error that was tricky to debug. And clearly I have not described it well enough :-)

I added the comment here, since this is the command producing the error. The root cause of the error was in the `sed` command (before/after) found in various places. In case I missed on, or a similar one may get added in the future, I added the comment here to save future developers some time. Instead of a lengthy comment, we could create a separate issue for it and mark it as addressed here.

But I would like to keep a clear path from error to resolution.

```Bash
sed -i.bak 's/newTag:.*$/newTag: '"$GIT_COMMIT_SHORT"'/' config/manager/kustomization.yaml. # before
sed -i.bak 's/newTag:.*$/newTag: "'${GIT_COMMIT_SHORT}'"/' config/manager/kustomization.yaml # after
```

Reply:
Thanks, I understand now! Personally, I'd prefer capturing that info in an Issue and linking to it with a shorter comment since it's a bit hard to read.

Reply:
I will clear it up and clean it up.

Reply:
I moved it to #473